Formula/l/logdy.rb

```diff
@@ -0,0 +1,19 @@
+class Logdy < Formula
+  desc "Web based real-time log viewer"
+  homepage "https://logdy.dev"
+  # uncomment with the next release
+  # url "https://github.com/logdyhq/logdy-core/archive/refs/tags/v0.9.0.tar.gz"
+  url "https://github.com/logdyhq/logdy-core/releases/download/v0.9.0/logdy-core-main.zip"
```

Comment:
what is the diff between the two?

Reply:
make sense, can you do tgz rather than zip? that would give better file compression ratio.